test/testcafe/README.md

```diff
@@ -23,7 +23,7 @@ Run all similar Fixtures that matches the pattern
 
 Run a single test
 
-- `yarn test -t testcafe chrome -t "should have editable fields"`
+- `yarn test -t testcafe chrome --test "should have editable fields"`
```

Comment:
We need to update Readme

1. It's not required to pass `chrome`, because it can default to what we have in rc file.
2. but we can have a single usage example on how to change the browser
   `yarn test -t testcafe <browser> --test "should have editable fields"`